samples/cellular/nrf_cloud_multi_service/README.rst

```diff
@@ -910,8 +914,8 @@ Next, for each device you wish to provision, execute the following command in th
 
 Where ``<certificate_serial>`` is your certificate's serial number in hex.
 
-Once the script creates the :file:`provision.csv` file, visit the `Provision Devices`_ page in the nRF Cloud portal to provision the device.
-Provisioning registers the device with nRF Cloud, allowing it to use any nRF Cloud services through CoAP, MQTT, or REST connectivity.
+Once the script creates the :file:`provision.csv` file, visit the **Devices** page in the nRF Cloud portal to onboard the device.
```

Comment:
The correct pages is not the `Devices` page, but, as Justin points out, the `Bulk Onboard Devices` page.

I also personally feel that being able to link to pages like this is valuable, so I am not on-board with removing the link, at least pending internal discussion